recipes/numpy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version="1.11.0" %}
+
+package:
+  name: numpy
+  version: {{ version }}
+
+source:
+  fn: numpy-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/n/numpy/numpy-{{ version }}.tar.gz
+  md5: bc56fb9fc2895aa4961802ffbdb31d0b
+  patches:
+    - osx-dist.patch          # [osx]
```

Comment:
I didn't write it.  I don't know.  Appears to force the -m32 or -m64 flag.  Maybe that goes wrong on OSX somehow?  Happy to try just removing the patch here...